### PR TITLE
1860: event_southern_forms text change

### PR DESCRIPTION
**Pull request by ryanespin**
Addresses #3374 

Changed text for SR endgame trigger to be less ambiguous.

### CODE REVIEW

File: lib/engine/game/g_1860.rb [hunk 1]
```diff
@@ -357,7 +357,7 @@ def event_relax_cert_limit!
       end
 
       def event_southern_forms!
-        @log << 'Southern Railway Forms; End of game triggered (via Nationalization).'
+        @log << 'Southern Railway Forms; End Game possible via Nationalisation.'
         @southern_formed = true
```
Comment (roseundy) [inline lib/engine/game/g_1860.rb:361]: I disagree somewhat - the game is going to end soon. If you want to warn that the game can still end via the bank breaking, you should put that in.

Comment (roseundy) [reply lib/engine/game/g_1860.rb:361]: I can live with "Southern Railway Form; Nationalization will be triggered when all players’ companies have at least one train"

Comment (ryanespin) [reply lib/engine/game/g_1860.rb:361]: Sure, so something like "Southern Railway Forms; End of game still possible via bank breaking".

If we want to include all end game condtions, maybe something like this: "Southern Railway Forms; End of game possible via Nationalisation, Bank Breaking, or Company hitting max stock value."

And lastly, should I go with nationalisation or nationalization?  The rulebook indicates nationalisation which is the UK spelling, but wasn't sure which to go with for the wording.

Comment (ryanespin) [reply lib/engine/game/g_1860.rb:361]: > I can live with "Southern Railway Forms; Nationalization will be triggered when all players’ companies have at least one train"

Updated! :)

**Review by tobymao — APPROVED** [review]

**Pull request merged**